### PR TITLE
DOC: add the 2022 interpolate project proposal

### PR DESCRIPTION
**Pull request by ev-br**
Add the interpolate project proposal, as discussed in the core-dev group, verbatim. Have to admit I've forgotten to add it here back in June, so fixing it up now.

### CODE REVIEW

File: proposals/use_scipy_funds/2022_001_clear_the_interpolate_backlog.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+#  Clearing up the `scipy.interpolate` backlog
+
+Proposal details:
+
+- Title: Clearing up the `scipy.interpolate` backlog
+- Author: Evgeni Burovski, Ralf Gommers
+- Date: Apr 25, 2022
+- Cost:
```
Comment (tupui) [inline proposals/use_scipy_funds/2022_001_clear_the_interpolate_backlog.md:8]: Intentionally omitted? Is this supposed to be reflected at some point here https://opencollective.com/scipy? Our fund handling is still nebulous to me.

**Review by tupui — COMMENTED** [review]
Thanks @ev-br!

File: proposals/use_scipy_funds/2022_001_clear_the_interpolate_backlog.md (hunk 1 above)
Comment (ev-br) [reply proposals/use_scipy_funds/2022_001_clear_the_interpolate_backlog.md:8]: Indeed, it will be reflected over there at some point.

Comment (tupui) [reply proposals/use_scipy_funds/2022_001_clear_the_interpolate_backlog.md:8]: Ok 👍 and about the amount itself?

**Review by tupui — APPROVED** [review]

**Comment by tupui** [comment]
Thanks @ev-br

**Pull request merged**